docs/web/data-api.mdx

```diff
@@ -0,0 +1,553 @@
+---
+title: 'Data API'
+description: 'REST API for MongoDB operations with Modelence'
+icon: 'database'
+---
+
+<Note>
+This tutorial assumes you've already [created a Modelence project](/quickstart) and [completed the setup](/setup). If you haven't done so, please complete those steps first.
+</Note>
+
+## What is MongoDB Data API
+
+An open-source API to read, write, and aggregate data in MongoDB. The application can be deployed to Modelence Cloud or to any other cloud provider.
+
+- **CRUD Operations**: Insert, find, update, and delete documents
+- **Advanced Querying**: Aggregation pipelines and complex queries
+- **Database Management**: Collection and index management
+- **Authentication**: API key-based security
+- **MongoDB Operations**: Direct access to MongoDB features
+
+## Sandbox
+
+Try the Data API live at [data-api-demo.modelence.app](http://data-api-demo.modelence.app/) to explore all endpoints and test operations without setup.
+
+## Project Setup
+
+### 1. Create a new application
+
+```bash
+npx create-modelence-app@latest ai-chat --template data-api
+```
+
+### 2. Connect to Modelenece Cloud
+
+1. Open [cloud.modelence.com](https://cloud.modelence.com/) create
+2. Create a new application and a local environment
+3. Click on Setting -> Set up
+4. Copy the command that connects to remote DB, it should looks like the following:
```

Comment:
During the setup steps there is no way to select remote DB. Do we need this step here?